test/file_acceptor/evidence.txt

```diff
@@ -1 +0,0 @@
-This is evidence.
```

Comment:
I thought these files were used for testing?

Reply:
I added them by mistake :-o

Reply:
They ended in by mistake, but now they should have gone.